Common/Python/PandasConverter.cs

```diff
@@ -166,7 +183,8 @@ public class PandasData
     {
         private readonly Symbol _symbol;
         private readonly List<DateTime> _timeIndex;
-        private readonly Dictionary<string, List<double>> _series;
+        private readonly Dictionary<string, List<double>> _doubleSeries;
```

Comment:
Not for this PR, but perhaps we should move `PandasData` to its own file in a separate change.